Feat: Feedback Component

Devendrakhatrii opened the pull request:
closes #15

isarojdahal commented:
https://github.com/isarojdahal/everydaykarmaa.com-assets/blob/main/images/feedbacks/

The above URL can be set to a const and reused on the values.

(If URL changes, so we can just update the const value)

isarojdahal commented:
@Ashmin-Bhujel  bhai, shall I merge this PR;
so you can do any missing updates on it . (as per our intial disucssion)

Devendrakhatrii commented:
> https://github.com/isarojdahal/everydaykarmaa.com-assets/blob/main/images/feedbacks/
> 
> The above URL can be set to a const and reused on the values.
> 
> (If URL changes, so we can just update the const value)

I've addressed the feedback and updated the code. Please let me know if any further changes are needed @isarojdahal daju!
